### ISSUE TITLE
Alexa Song stopped on phone lock or notification received on phone, while phone is connected to Bluetooth

### ISSUE THREAD

**Issue by mohitbansal19** [alexa/avs-device-sdk#1470]
## Briefly summarize your issue:
  
With AVS-SDK 1.14.0, it is observed that song played with Alexa stopped on phone lock or on notification while phone connected with Bluetooth 

## What is the expected behavior?
Alexa song should not stop on phone lock/unlock 

## What behavior are you observing?
Song played with Alexa, stopped

## Provide the steps to reproduce the issue, if applicable:
 
1. Configure Alexa
2. Say "Alexa" and then ask alexa to play some song
3. Connect phone with Bluetooth
4. Lock/unlock or receive notification on phone while song is playing
5. Song playing with Alexa will stop

## Tell us about your environment:

AVS SDK Version 1.14.0 Cross Compiled to run on below Hardware.
PulseAudio
Bluetooth
 
### What version of the AVS Device SDK are you using?
1.14.0

### Tell us what hardware you're using:
Other: Hardware: Samsung Artik 305 Secure Board
ARH: ARMv7

### Tell us about your OS (Type & version):
Linux, Ubuntu 16.04
Linux artik 4.4.71-UNRELEASED #1 SMP PREEMPT Tue Aug 20 18:39:35 IST 2019 armv7l armv7l armv7l GNU/Linux
[alexa_isssue_1.log](https://github.com/alexa/avs-device-sdk/files/3524381/alexa_isssue_1.log)

**Comment by kclchan** [comment]
Hi @mohitbansal19,

Thanks for bringing this issue to our attention.  We have investigated the logs and found the root cause of the problem.  The issue is due to the Bluetooth kicking out the AudioPlayer from the Content channel, so that's why the audio playback was not able to resume. 

```
2019-08-21 08:28:06.601 [ 29] 1 FocusManager:acquireChannel:channelName=Content,interface=Bluetooth
2019-08-21 08:28:06.603 [ 11] 0 AudioPlayer:onFocusChanged:newFocus=NONE
```

We have created an internal ticket to get this issue fix in a future release of the SDK.  Thanks again!

Thanks,
Kenneth

**Comment by mohitbansal19** [comment]
Hi @kclchan
 
  Thanks for your reply on this issue.

Is there any workaround for this issue for now?

Thanks
Mohit Bansal

**Comment by caleighatamazon** [comment]
Hi there, sorry for our late response on this. Are you still having problems?

**Comment by mohitbansal19** [comment]
Hi,
  In avs-1.14 this issue exists. We did not try this with latest build, 1.16. 
Did you resolve this problem in latest 1.16 build?

**Comment by mohitbansal19** [comment]
Hi,

   Is this issue has been fixed in latest version i.e 1.17.0?
Please give your confirmation.

Thanks
Mohit Bansal

**Comment by kclchan** [comment]
Hi @mohitbansal19, this issue is not fixed in v1.17.0 yet.  We have prioritized this issue and will be fixed in the near future.

**Comment by kclchan** [comment]
Hi @mohitbansal19, we have just released v1.18 that contains the fix for this issue.  Thanks!
